{"uid":"4aa21176c1333b6a","name":"Usuário autenticado e associado ao perfil","fullName":"Validação de Usuário: Usuário autenticado e associado ao perfil","historyId":"52f160ff65549f9decdc744885d8065f","time":{"start":1747850319426,"stop":1747850319709,"duration":283},"status":"failed","statusMessage":"AssertionError: O cenário de cadastro de usuário não foi executado com sucesso antes deste passo.\n","statusTrace":"  File \"C:\\Automação de Testes\\neo_liberalizados-automacao\\venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Automação de Testes\\neo_liberalizados-automacao\\venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\test_tela_cadastro_usuario_steps.py\", line 37, in step_usuario_foi_cadastrado\n    page.validar_usuario_cadastrado(context)\n  File \"C:\\Automação de Testes\\neo_liberalizados-automacao\\features\\pages\\tela_cadastro_usuario_pages.py\", line 122, in validar_usuario_cadastrado\n    assert False, (\n           ^^^^^\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"AssertionError: O cenário de cadastro de usuário não foi executado com sucesso antes deste passo.\n","statusTrace":"  File \"C:\\Automação de Testes\\neo_liberalizados-automacao\\venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Automação de Testes\\neo_liberalizados-automacao\\venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\test_tela_cadastro_usuario_steps.py\", line 37, in step_usuario_foi_cadastrado\n    page.validar_usuario_cadastrado(context)\n  File \"C:\\Automação de Testes\\neo_liberalizados-automacao\\features\\pages\\tela_cadastro_usuario_pages.py\", line 122, in validar_usuario_cadastrado\n    assert False, (\n           ^^^^^\n","steps":[{"name":"Given que o usuário está logado no sistema","time":{"start":1747850319426,"stop":1747850319573,"duration":147},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentStep":false,"hasContent":false,"shouldDisplayMessage":false,"attachmentsCount":0},{"name":"Given que o usuário foi cadastrado no sistema","time":{"start":1747850319573,"stop":1747850319708,"duration":135},"status":"failed","statusMessage":"AssertionError: O cenário de cadastro de usuário não foi executado com sucesso antes deste passo.\n","statusTrace":"  File \"C:\\Automação de Testes\\neo_liberalizados-automacao\\venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Automação de Testes\\neo_liberalizados-automacao\\venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\test_tela_cadastro_usuario_steps.py\", line 37, in step_usuario_foi_cadastrado\n    page.validar_usuario_cadastrado(context)\n  File \"C:\\Automação de Testes\\neo_liberalizados-automacao\\features\\pages\\tela_cadastro_usuario_pages.py\", line 122, in validar_usuario_cadastrado\n    assert False, (\n           ^^^^^\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentStep":false,"hasContent":true,"shouldDisplayMessage":true,"attachmentsCount":0},{"name":"Then o usuário deve ser exibido na tela de usuários","time":{"start":1747850319709,"stop":1747850319709,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentStep":false,"hasContent":false,"shouldDisplayMessage":false,"attachmentsCount":0}],"attachments":[],"parameters":[],"stepsCount":3,"attachmentStep":false,"hasContent":true,"shouldDisplayMessage":false,"attachmentsCount":0},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"feature","value":"Validação de Usuário"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[]}],"tags":[]},"source":"4aa21176c1333b6a.json","parameterValues":[]}